src/ui-lib/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss

```diff
@@ -56,12 +56,7 @@ body > header {
     display: block;
     max-height: 33px; // Otherwise IE11 makes it 150px tall :-(
     fill: $grav-co-bg;
-
-    // Bump up logo size
-    @media (min-width: gravy-breakpoint(medium)) {
-      height: 2.5rem;
```

Comment:
The current solution makes things a bit wonky on smaller screens - the SVG _container_ fills 100% of the space right of the hamburger button, but the logo is centered within that, so it is no longer flush with the left edge of the header:

![image](https://user-images.githubusercontent.com/16718916/39533402-d56ac2e4-4e26-11e8-9f21-841f2584a134.png)

To resolve this, re-instate the media query you removed, and move the `width: 100%;` within it (replacing the `height: 2.5.rem` that was there before).

Reply:
So, all together, it should look like this:
```sass
  .grav-c-logo {
    display: block;
    max-height: 33px; // Otherwise IE11 makes it 150px tall :-(
    fill: $grav-co-bg;

    // Bump up logo size
    @media (min-width: gravy-breakpoint(medium)) {
      width: 100%;
      max-height: none;
    }
  }
```